script/Ciel/Day 9/09_01_CIEL09_4.txt

```diff
@@ -381,42 +451,58 @@ From what I heard, the longest time she's ever taken was one year, during which
 [sha:27b25b4c013049ac675353f096153f1ba2f9f7c5]{
 -- Page 405, Offset 23179.
 -- で、あの子が選んだ手段は排水溝からの侵入と潜伏。下水路すら死徒にとっては血管みたいなものだったけど、その血管の中を流れる汚物にまでは感覚が届かなかったのね」
-That's why she chose to enter from a drainage ditch and lie in wait. The Dead Apostle's range went as far as the sewers, and he considered them similar to his blood vessels, but his perception didn't extend to the filth drifting through them."
+That's why she chose to infiltrate the city through a culvert and hide herself in the sewers.
+ Those pipes may have been like blood vessels to the Dead Apostle,
+ but his senses didn't extend to the filth that drifted through those veins."
 }
 [sha:8dccb5c60c32dd755d24a426edbb2338c6940e0e]{
 -- Page 406, Offset 23180.
 -- 「秘蹟も魔術も使えば感知される状況だったから、あの子は黒鍵の<柄|え>を胃に呑み込んで下水に入っていった。
-"Since using a Sacrament or magecraft would make her presence known, she swallowed the handle of a Black Key before entering the sewers.
+// Parsing 柄 as just the handle doesn't make much sense here - assuming it's more like the stalk/length of an _entire_ black key?
+// Hard to do much stabbing with just the hilt, surely
+"Since using any sacraments or magecraft would give away her presence,
+ she swallowed the entire length of a black key before entering the sewers.
 }
 [sha:64539c7ba6d7e208e70ec1711a5d9a028cf7a33a]{
 -- Page 406, Offset 23181.
 -- この国の整備された下水じゃない。汚物でつまったヨーロッパの下水路で、体を汚水にひたして、顔だけを水面に出してギリギリの呼吸をして、ただ機会を待ち続ける一年間！」
-Those sewers weren't like this country's well-maintained drainage systems. She bode her time for a whole year, barely able to breathe. Only her head was above the water, while the rest of her body was submerged in the black water of some filthy European sewer!"
+// Wonder if she came into contact with the fatberg (rip)
+Those sewers weren't as well-maintained as the ones in this country.
+ She spent an entire year biding her time with her body immersed in the effluent of some European sewer,
+ only her face above water so that she could breathe,
+ waiting for her chance to strike."
```

Comment:
Like the other suggestion here, it just strikes me as way too many commas for one sentence

Reply:
Yeah, my suggestion is probably not the best, all things considered, but this still needs major restructuring to retain the feeling of the original.

The argument for cutting it up into several sentences feels like its simply because that's a simpler way to write. The sequence of events are clearly highly interconnected, so it feels like a longer sentence would be more conductive to actually getting across the sheer persistence of Ciel.

If there's one part that can be singled out into its own sentence, then its the first clause, since it highlights the difference of the "sewers in this country <-> sewers in Europe".

>This wasn't the kind of well-maintained sewer you can find in this country.
>It was a European sewer, chocked with filth.
>She had immersed her body in that sewage,
>with nothing but her face above water,
>barely even breathing,
>bidding her time for a whole year!